scripts/zones/Dynamis-Valkurm/mobs/Adamantking_Image.lua

```diff
@@ -35,6 +29,10 @@ function onMobDeath(mob, player, isKiller)
                 effect:setDuration(effect:getDuration() * 1000 + 20 * 60 * 1000)
                 player:messageSpecial(DYNAMIS_TIME_EXTEND,20)
             end
+            DisallowRespawn(mob:getID(), true)
+            local newTEMob = math.random(0,3) -- Randomly select new TE5 mob
```

Comment:
probably condense this into local `newTEMob = TE5_RANDOM[math.random(#TEE5_RANDOM)]`